odl_video/settings.py

```diff
@@ -104,6 +104,9 @@ def get_var(name, default):
 # Application definition
 
 INSTALLED_APPS = [
+    'ui.apps.UIConfig',
+    'cloudsync.apps.CloudSyncConfig',
+    'dj_elastictranscoder',
```

Comment:
`ui` in particular had to be before `django.contrib.admin` to override the admin logout page.  I decided to also place `cloudsync` and `dj_elastictranscoder` above the standard django apps to preemptively avoid similar issues.  I ran unit tests and manually tested the app to see if there were any unintended side effects but did not see any.  If you prefer I can put `cloudsync` and `dj_elastictranscoder` back in their original order.